tests/ci/cdk/run-cdk.sh

```diff
@@ -25,31 +25,45 @@ shift
 
 # Export other environment variables.
 export DATE_NOW="$(date +%Y-%m-%d-%H-%M)"
-export ECR_LINUX_AARCH_REPO_NAME="aws-lc-test-docker-images-linux-aarch"
-export ECR_LINUX_X86_REPO_NAME="aws-lc-test-docker-images-linux-x86"
-export ECR_WINDOWS_REPO_NAME="aws-lc-test-docker-images-windows"
-export S3_FOR_WIN_DOCKER_IMG_BUILD="windows-docker-images-${DATE_NOW}"
+export ECR_LINUX_AARCH_REPO_NAME="aws-lc-docker-images-linux-aarch"
+export ECR_LINUX_X86_REPO_NAME="aws-lc-docker-images-linux-x86"
+export ECR_WINDOWS_REPO_NAME="aws-lc-docker-images-windows"
+export AWS_LC_S3_BUCKET_PREFIX="aws-lc-windows-docker-image-build"
+export S3_FOR_WIN_DOCKER_IMG_BUILD="${AWS_LC_S3_BUCKET_PREFIX}-${DATE_NOW}"
 export WIN_EC2_TAG_KEY="aws-lc"
-export WIN_EC2_TAG_VALUE="windows-docker-img-${DATE_NOW}"
+export WIN_EC2_TAG_VALUE="aws-lc-windows-docker-image-build-${DATE_NOW}"
 export WIN_DOCKER_BUILD_SSM_DOCUMENT="windows-ssm-document-${DATE_NOW}"
 
 # Functions
+function delete_s3_buckets() {
+  aws s3api list-buckets --query "Buckets[].Name" | jq '.[]' | while read -r i; do
+    # Delete the bucket if its name uses AWS_LC_S3_BUCKET_PREFIX.
+    if [[ ${i} == *"${AWS_LC_S3_BUCKET_PREFIX}"* ]]; then
```

Comment:
Do we need to precede this prefix  with `*`? If these buckets always start with this prefix, we only care about what comes after right?

Reply:
Removed.